plugins/org.jkiss.dbeaver.cmp.simple.ui/OSGI-INF/l10n/bundle_pt_BR.properties

```diff
@@ -1,4 +1,7 @@
-command.org.jkiss.dbeaver.core.compare.simple.name=Compara\u00E7\u00E3o de estrutura simples
-command.org.jkiss.dbeaver.core.compare.simple.description=Compara\u00E7\u00E3 simples do banco objeto e estrutura
+# Help in keeping DBeaver Brazilian Portuguese translation standardized:
+# https://github.com/brlarini/dbeaver_l10n_pt-BR_guidelines
```

Comment:
We don't want to add links to outside dbeaver repositories. Maybe I could add a wiki page with your text and change your references to references to our wiki. Are you okay with this?